src/visor/readable.go

```diff
@@ -127,6 +127,8 @@ type ReadableTransactionOutput struct {
 type ReadableTransactionInput struct {
 	Hash    string `json:"uxid"`
 	Address string `json:"owner"`
+	Coins   string `json:"coins"`
+	Hours   uint64 `json:"hours"`
```

Comment:
hours should always be strings

Reply:
aren't hours like elapsed time?
they are stored as `uint64` everywhere else

Reply:
API policy going forward is to using strings for hours